jobs/README.md

```diff
@@ -7,6 +7,25 @@ commit, git push, and create a pull request on GitHub.
 __Be advised that Job Openings older than 3 months will be removed for the sake
 of keeping the board up to date. Feel free to repost.__
 
+## Web Developer @ [infomen](https://www.linkedin.com/company/infomen-gr/)
```

Comment:
Is there a relevant job link at https://www.infosupport.gr/el/%CE%B5%CF%85%CE%BA%CE%B1%CE%B9%CF%81%CE%AF%CE%B5%CF%82-%CE%BA%CE%B1%CF%81%CE%B9%CE%AD%CF%81%CE%B1%CF%82 to add a link here?